CrewChiefV4/GameState/GameStateMapper.cs

```diff
@@ -188,7 +188,8 @@ protected int getRacePosition(String driverName, int oldPosition, int newPositio
             }
             if (newPosition < 1)
             {
-                Console.WriteLine("Can't update position to " + newPosition);
+                //dont warn about this it happens a lot in iracing on session end. 
+                //Console.WriteLine("Can't update position to " + newPosition);
```

Comment:
Maybe instead of commenting this out, it is worth keeping it but checking if game != iRacing?

Reply:
Yes, i think it's worth adding if-not-iracing here. Not sure how useful the message is but it might help chase down some problem